components/Logo/LogoLockup.styles.ts

```diff
@@ -1,8 +1,8 @@
-export const root = 'no-underline inline-block font-normal';
-export const contentWrapper = 'flex-col sm:flex-row items-start sm:items-center';
```

Comment:
On the live site, the logo stacks vertically only in the local footer - we were going to have it match the masthead (logo look the same for all breakpoints with the bar after the Stanford wordmark), but we never got to it, so I'm doing that here.